plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestOracleTaskFailureRecoveryWithPoolEnabledTest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.oracle;
+
+import io.trino.operator.RetryPolicy;
+
+public class TestOracleTaskFailureRecoveryWithPoolEnabledTest
```

Comment:
How much do those take? I am not sure if we need full pool/no-pool task/query coverage

Reply:
Are suggest we could remove this(the pool enabled) test case?
> How much do those take?

I am not sure how to measure the cost, but I have done tests both situation two times on my local:

- Without pool enabled: times 13min42, 14min10, highest cpu usage consume (on docker container) around 2 cores, memory consumes around 500MB
- With pool enabled: times 13min20, 15min39, cpu and memory is similar to the pool disabled

Reply:
@losipiuk Removed the pool enabled test cases. cc @hashhar